OpenSourcePolitics/decidim-app
# feat: Add extended socio demo ah

#### :tophat: Description
Add the extended socio demo authorization handler gem 

#### Testing
* Log in as user
* Go to accoun settings
* Go to authorizations
* See "Additionnal informations" AH
